src/pages/Patients.jsx

```diff
@@ -51,33 +60,75 @@ const Patients = () => {
       field: "actions",
       headerName: "Actions",
       width: "310px",
-      cellRenderer: PatientsButtonsRenderer
+      cellRenderer: (params) => (
+        <PatientsButtonsRenderer
+          data={params.data}
+          onDelete={() => {
+            handleDelete(params.data.patientId);
+          }}
+        />
+      )
     }
   ];
+  const handleClose = () => {
+    setToastData({
+      openToast: false
```

Comment:
You will have to use the spread operator here so that the other properties can be copied over when you make the change to the open or closed status 

Something like this ..
SetToastData ({...toastData, openToast:true})